examples/cpp/run_model.cpp

```diff
@@ -0,0 +1,67 @@
+#include <torch/script.h>
+#include <torch/torch.h>
+#include <cstring>
+#include <iostream>
+
+#ifdef _WIN32
+#include <torchvision/vision.h>
+#endif // _WIN32
```

Comment:
I don't fully understand yet why this is required on Windows, but this is for another PR